metron-interface/metron-alerts/e2e/alerts-list/meta-alerts/meta-alert.e2e-spec.ts

```diff
@@ -137,14 +135,16 @@ describe('Test spec for meta alerts workflow', function() {
     expect(await metaAlertPage.getAvailableMetaAlerts()).toEqualBcoz('e2e-meta-alert (22)', 'Meta alert should be present');
     await metaAlertPage.selectRadio();
     await metaAlertPage.addToMetaAlert();
-    expect(await tablePage.getCellValue(0, 2, '(22')).toContain('(23)', 'alert count should be incremented');
+    // FIXME: line below will fail because the following: https://hortonworks.jira.com/browse/BUG-106815
```

Comment:
Can you make this and the other FIXMEs reference the Apache Jira (https://issues.apache.org/jira/browse/METRON-1654) instead of the Hortonworks Jira?